.github/workflows/release.yml

```diff
@@ -84,19 +88,23 @@ jobs:
         with:
           token: ${{ secrets.GH_TOKEN }}
           ref: v${{ needs.publish.outputs.release_version }}
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          build-scan-publish: true
+          build-scan-terms-of-use-url: "https://gradle.com/terms-of-service"
+          build-scan-terms-of-use-agree: "yes"
       - name: Nexus Staging Close And Release
-        uses: gradle/gradle-build-action@v2
+        run: >
+          ./gradlew findSonatypeStagingRepository
+          releaseSonatypeStagingRepository 
         env:
           DEVELOCITY_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
```

Comment:
`DEVELOCITY_ACCESS_KEY` is not necessary here. A short-lived access key It is provided by `setup-gradle` thanks to `with: develocity-access-key: ...`